sale_packaging_default/views/sale_order_view.xml

```diff
@@ -36,6 +36,33 @@
                     position="move"
                 />
             </xpath>
+
+            <xpath
+                expr="//field[@name='order_line']/tree/field[@name='product_packaging_id']"
+                position="before"
+            >
+                <field name="require_packaging" invisible="1" />
+            </xpath>
+
+            <!-- Make packaging mandatory when the product is not allow to divide the packaging -->
+            <xpath
+                expr="//field[@name='order_line']/tree/field[@name='product_packaging_id']"
+                position="attributes"
+            >
+                <attribute
+                    name="attrs"
+                >{'required': [('require_packaging', '=', True)]}</attribute>
+            </xpath>
+
+            <!-- Make packaging qty readonly when the product is not allow to divide the packaging -->
+            <xpath
+                expr="//field[@name='order_line']/tree/field[@name='product_packaging_qty']"
+                position="attributes"
+            >
+                <attribute
+                    name="attrs"
+                >{'readonly': [('require_packaging', '=', True)]}</attribute>
+            </xpath>
```

Comment:
issue: so, we require packaging, but we don't allow to set how many? It doesn't make sense:
![image](https://github.com/user-attachments/assets/ef105059-68bf-4d54-8c52-a7f039cfc023)

Instead, please do the opposite thing. The field that you should make readonly is `product_uom_qty`. That way, the user is not allowed to break the packaging. They're only allowed to increase or decrease whole packaging sets.